androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/WootricRemoteRequestTask.java

```diff
@@ -24,7 +24,7 @@
     static final String REQUEST_TYPE_GET = "GET";
 
     protected static final String API_ENDPOINT = "https://api.wootric.com";
-    protected static final String SURVEY_ENDPOINT = "http://survey.wootric.com/";
+    protected static final String SURVEY_ENDPOINT = "http://survey.wootric.com";
```

Comment:
cause it's also here:
`protected static final String ELIGIBLE_URL = SURVEY_ENDPOINT + "/eligible.json";`
So the request was like this:
`http://survey.wootric.com//eligible.json?`